### PR TITLE
SAT: Replace validate_output_from_all_streams with empty_streams param

### PR DESCRIPTION
**Pull request by keu**
## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## Pre-merge Checklist
Expand the checklist which is relevant for this PR. 

<details><summary> <strong> Connector checklist </strong> </summary>
<p>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Secrets are annotated with `airbyte_secret` in the connector's spec
- [ ] Credentials added to Github CI if needed and not already present. [instructions for injecting secrets into CI](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#using-credentials-in-ci). 
- [ ] Unit & integration tests added as appropriate (and are passing)
    * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
    * Community members can skip this, Airbyters will run this for you. 
- [ ] Code reviews completed
- [ ] Documentation updated 
    - [ ] `README.md`
    - [ ] `docs/SUMMARY.md` if it's a new connector
    - [ ] Created or updated reference docs in `docs/integrations/<source or destination>/<name>`.
    - [ ] Changelog in the appropriate page in `docs/integrations/...`. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md` contains a reference to the new connector
    - [ ] Build status added to [build page](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/builds.md)
- [ ] Build is successful
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-a-connector)
- [ ] No major blockers
- [ ] PR merged into master branch
- [ ] Follow up tickets have been created
- [ ] Associated tickets have been closed & stakeholders notified
</p>
</details>

<details><summary> <strong> Connector Generator checklist </strong> </summary>
<p>
   
- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed.
</p>
</details>

### CODE REVIEW

File: airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py [hunk 1]
```diff
@@ -123,6 +123,52 @@ def primary_keys_for_records(streams, records):
...
+        all_streams = set(stream.stream.name for stream in configured_catalog.streams)
+        streams_with_records = set(counter.keys())
+        streams_without_records = all_streams - streams_with_records
+
+        streams_without_records = streams_without_records - allowed_empty_streams
+        assert not streams_without_records, f"All streams should return some records, streams without records: {streams_without_records}"
+
+    def _validate_expected_records(self, records, expected_records, flags):
+        """
+        We expect
```
Comment (keu) [inline airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:156]: probably lost connection with my head somewhere here, will fix

File: airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py [hunk 2]
```diff
@@ -123,6 +123,52 @@ def primary_keys_for_records(streams, records):
 
 @pytest.mark.default_timeout(5 * 60)
 class TestBasicRead(BaseTest):
+    @staticmethod
+    def _validate_schema(records, configured_catalog):
```
Comment (sherifnada) [inline airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:127]: unit tests?

**Review by sherifnada — APPROVED** [review]
LGTM but can you add unit tests?

File: airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py (hunk 2 above)
Comment (keu) [reply airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/tests/test_core.py:127]: unfortunately not for the whole test, but we have a test for validate_schema

**Comment by sherifnada** [comment]
@eugene-kulak I think this is breaking integration tests for other connectors, could you make sure we do a PR to remove the deleted property from their YAMLs?

**Comment by keu** [comment]
@sherifnada on it ATM

**Pull request merged**